build_projects/Microsoft.DotNet.Cli.Build.Framework/project.json

```diff
@@ -1,5 +1,6 @@
 {
   "version": "1.0.0-*",
+
```

Comment:
These changes can probably be reverted.  The next time update-dependencies runs it will remove the blank lines anyway.